paddle/fluid/framework/details/computation_op_handle.h

```diff
@@ -45,6 +56,10 @@ struct ComputationOpHandle : public OpHandleBase {
   std::unique_ptr<OperatorBase> op_;
   Scope *scope_;
   platform::Place place_;
+  size_t scope_idx_;
+#ifdef PADDLE_WITH_CUDA
+  bool is_lock_and_record_event_free_{false};
```

Comment:
Maybe we can just remove the macro. It is annoying and change the ABI between GPU/CPU

Reply:
Done. Thanks.